typing/typemod.ml

```diff
@@ -331,6 +335,7 @@ let map_ext fn exts rem =
    making them abstract otherwise. *)
 
 let rec approx_modtype env smty =
+  (* Need to process warning attributes here ?? *)
```

Comment:
Yes, indeed.  This would fall in the scope of https://github.com/ocaml/ocaml/pull/1191/files I think.